contracts/SavingAccount.sol

```diff
@@ -31,7 +31,7 @@ contract SavingAccount is Initializable, InitializableReentrancyGuard, Constant,
     event WithdrawCOMP(address beneficiary, uint256 amount);
 
     modifier onlySupportedToken(address _token) {
-        if(!Utils._isETH(address(globalConfig), _token)) {
+        if(_token!=ETH_ADDR) {
```

Comment:
Pls fix the spacing format, for example
`x != y`

Reply:
fixed